<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GItemDisplayWidget</class>
 <widget class="QWidget" name="GItemDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>16</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>       </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelValue">
     <property name="text">
      <string>         </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSuffix">
     <property name="text">
      <string>           </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
